Source/FieldSolver/WarpXPushFieldsEM.cpp

```diff
@@ -11,12 +11,11 @@
 #include "BoundaryConditions/PML.H"
 #include "Evolve/WarpXDtType.H"
 #include "FieldSolver/FiniteDifferenceSolver/FiniteDifferenceSolver.H"
-#if defined(WARPX_DIM_RZ) || defined(WARPX_USE_PSATD)
+#if defined(WARPX_USE_PSATD)
```

Comment:
Same here as above, that only one should be imported. Also, SpectralFieldData should be treated the same - it's strange that only SpectralFieldData.H is imported and not also SpectralFieldDataRZ.H.

Reply:
Thank you @dpgrote 
This is fixed now